docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +126,383 @@ microservice reports itself as ready.
 |Treats the instance as still starting up; does not check liveness or readiness until the startup probe reports success or times out according to its configuration.
 |===
 
-=== Built-in and Custom Health Checks
 
-==== Built-in Health Checks
-Helidon provides built-in, default checks for each endpoint.
-The built-in liveness checks include various environmental values, such as whether the JVM has detected deadlocks
-or whether there is sufficient heap space. The built-in readiness and startup checks always report `UP`.
+== Configuration
+
+=== Custom Health Root Path and Port
+
+You can specify a custom port and root context for the root health endpoint path.
+However, you cannot use different ports, such as http://localhost:8080/myhealth and http://localhost:8081/myhealth/live.
+Likewise, you cannot use different paths, such as http://localhost:8080/health and http://localhost:8080/probe/live.
+
+The example below will change the root path.
+
+[source,yaml]
+.Create a file named `application.yaml` in the `resources` directory with the following contents:
+----
+health:
+  web-context: "myhealth"  // <1>
+----
+<1> The web-context specifies a new root path for the health endpoint.
+
+[source,bash]
+.Build and run the application, then verify that the health endpoint is using the new `/myhealth` root:
+----
+> curl http://localhost:8080/myhealth
+> curl http://localhost:8080/myhealth/live
+> curl http://localhost:8080/myhealth/ready
+> curl http://localhost:8080/myhealth/started
+----
+
+== Reference
+
+Configuration properties, values and description
+
+|===
+|Property | Values | Description
+
+|health:web-context
+|`String`
+| The endpoint name on which the health context is propagated
+
+|===
+
+
+== Examples
+
+Generate Helidon MP Quickstart project following these <<Instructions>>
+
+=== Using the Built-In Health Checks
+
+Helidon has a set of built-in health checks that are automatically enabled to report various
+health check statuses that are commonly used:
```

Comment:
Pre-existing error. Remove "automatically enabled." We earlier mentioned the built-in health checks and the extra dependency developers must add to include them. They are not "automatic" because they will not be present unless the developer adds that dependency.